lib/jsdom/living/events/InputEvent-impl.js

```diff
@@ -6,14 +6,15 @@ const InputEventInit = require("../generated/InputEventInit");
 
 // https://w3c.github.io/uievents/#interface-inputevent
 class InputEventImpl extends UIEventImpl {
-  initInputEvent(type, bubbles, cancelable, data, isComposing) {
+  initInputEvent(type, bubbles, cancelable, data, isComposing, inputType) {
```

Comment:
It looks like this function is never used or exposed (since it's not in the IDL), so we can delete it. I'll push a commit to do that.